CefSharp.Wpf/Rendering/InteropBitmapInfo.cs

```diff
@@ -31,14 +32,24 @@ public override void ClearBitmap()
 
         public override void Invalidate()
         {
-            Bitmap.Invalidate();
+            if (Bitmap != null)
+            {
+                Bitmap.Invalidate();
+            }
         }
 
         public override BitmapSource CreateBitmap()
         {
             var stride = Width * BytesPerPixel;
 
-            Bitmap = (InteropBitmap)Imaging.CreateBitmapSourceFromMemorySection(FileMappingHandle, Width, Height, PixelFormat, stride, 0);
+            if (FileMappingHandle == IntPtr.Zero)
+            {
+                ClearBitmap();
```

Comment:
`CreateBitmap()` is usually called after the bitmap has already been cleared. In the scenario your trying to handle hear is it possible for `bitmap != null`?

Ideally would fail fast, have a code block like this as the first operation.
```c#
// Unable to create bitmap without valid File Handle (Most likely control is being disposed)
if (FileMappingHandle == IntPtr.Zero)
{
  return null;
}
```